esmvalcore/preprocessor/_time.py

```diff
@@ -711,3 +754,73 @@ def timeseries_filter(cube, window, span,
                                weights=wgts)
 
     return cube
+
+
+def resample_hours(cube, hours):
+    """
+    Converts x-hourly data to y-hourly by eliminating extra timesteps.
+
+    Converts x-hourly data to y-hourly (y > x) by eliminating the extra
```

Comment:
Please add this (y > x) as a check or assertion

Reply:
Done